src/Reversi/Board.fs

```diff
@@ -1,34 +1,52 @@
 namespace FableReversi.Reversi
 
+module Bitwise =
+    let inline pos (x:int) (y:int) = x + 8 * y
+
+    let inline getXY (pos:int) = pos % 8, pos / 8
+
+    let inline setStone (pos:int) (board:uint64) =
+        board ||| (1UL <<< pos)
+
+    let inline removeStone (pos:int) (board:uint64) =
+        board &&& (~~~ (1UL <<< pos))
+
+    let inline isSet (pos:int) (board:uint64) =
+        ((board >>> pos ) &&& 1UL) = 1UL
+
+    let countStones (board:uint64) =
+        let mutable count = 0UL
```

Comment:
Should return an int so we can safely subtract values later (this broke the minimax player.)